### DISCUSSION TITLE
reversible model of infectious disease can be implemented in POMP

### DISCUSSION THREAD

**Discussion by manal12341234** [kingaa/pomp#162] [Q&A] [answered]
Could we simulate a reversible model for infectious model via pomp package especially the part " A binomial approximation with exponential transition probabilities"? in order to use the "rprocess" and other useful functions which needs this information?
If so how to implement this time step for the attached file. 

![SEIRDV without demography](https://user-images.githubusercontent.com/70869344/137637557-bbca27b6-b1bc-4d56-bb6f-586f8490f00c.png)

Thanks and regards

**Comment by kingaa** [comment]
In what sense is this model "reversible'?

**Comment by manal12341234** [comment]
It is to simulate the inefficacy of vaccine (sigma=0.15) and in case of loss of immunity  after 9 months(delta=1/(9*30) .
Thanks for your response
Regards

**Comment by kingaa** [comment]
Of course, such compartmental models can be implemented in **pomp**.  There are many examples in the literature.  See the [bibliography](https://kingaa.github.io/pomp/biblio.html) for some of these.

**Reply by manal12341234** [reply]
Really many thanks for your feedback and response. Sure I will go through them.
Could you please advise me regarding the second question I posted ?

REGARDS

**Comment by kingaa** [comment, marked as answer]
What you seek to do is straightforward, but not without its complexities.  I haven't the time to give you private tutelage.  My advice is to follow the course[*Simulation-based Inference for Epidemiological Dynamics*](https://kingaa.github.io/sbied/) and to study a text, such as [Keeling & Rohani, *Modeling Infectious Diseases in Humans and Animals*](http://www.modelinginfectiousdiseases.org/).
